randovania/exporter/patch_data_factory.py

```diff
@@ -90,11 +92,19 @@ def create_data(self, custom_metadata: PatcherDataMeta | None = None) -> dict:
         :return: The patcher data, with the randovania metadata included, as a dict.
         """
 
-        randovania_meta = custom_metadata or self.create_default_patcher_data_meta()
+        with sentry_sdk.isolation_scope() as scope:
+            scope.add_attachment(
+                json.dumps(self.description.as_json(force_spoiler=True)).encode("utf-8"),
+                filename="rdvgame.json",
+                content_type="application/json",
+                add_to_transactions=True,
+            )
 
-        game_data = self.create_game_specific_data(randovania_meta)
-        json_delta.patch(game_data, self.patches.custom_patcher_data)
-        game_data["_randovania_meta"] = randovania_meta
+            randovania_meta = custom_metadata or self.create_default_patcher_data_meta()
```

Comment:
Not sure if this is too much of a nitpick, but should this be indented into the `with` block? At the very least, not indenting it would make a cleaner commit on squash merge. To me, that would also communicate what the code does slightly more clearly.

Reply:
i suppose the indent might not actually be necessary here.
if you want to start a transaction for measuring purposes, then all the code has to be inside of the transaction scope.
But since we're only attaching a file to the current scope, we can probably get rid of it.